src/main/java/de/samply/icd10dictionary/model/IcdCode.java

```diff
@@ -1,68 +1,16 @@
 package de.samply.icd10dictionary.model;
 
-public class IcdCode {
-
-    private String code;
-    private String kind;
-    private String display;
-    private String definition;
-    private String parentCode;
-    private String childCodes;
-
-    public IcdCode(String code, String kind, String display, String definition, String parentCode, String childCodes) {
-        this.code = code;
-        this.kind = kind;
-        this.display = display;
-        this.definition = definition;
-        this.parentCode = parentCode;
-        this.childCodes = childCodes;
-    }
-
-    public String getCode() {
-        return code;
-    }
-
-    public void setCode(String code) {
-        this.code = code;
-    }
-
-    public String getKind() {
-        return kind;
-    }
-
-    public void setKind(String kind) {
-        this.kind = kind;
-    }
+import lombok.AllArgsConstructor;
```

Comment:
Why don't we just use the HAPI Model Classes?

Reply:
To keep this prject slim and avoid unnecessary dependencies. But you can write an issue and we can prioratize it.

Reply:
Ok lombok is really nice. Didn't know it. So may be even better than use the overly blown HAPI Model Classes.